src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.head {
+  width: 100%;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.inner__container {
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: rgb(255, 255, 255);
+  position: relative;
+  bottom: 1px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px;
+  margin-right: 0;
+  margin-left: 50px;
+}
+
+.nav__link {
+  display: flex;
+  text-transform: uppercase;
+  padding: 0;
+  margin: 0;
+  margin-right: 50px;
+  list-style: none;
+}
+
+.refer:link {
+  text-decoration: none;
+  color: black;
+  position: relative;
+  padding: 22.5px 0;
+}
+
+.refer:visited {
+  color: #00acdc;
+}
+
+.refer:hover {
+  color: #00acdc;
+}
+
+.nav__link :first-child .refer {
+  margin: 0 10px 0 0;
+}
+
+.nav__link :nth-child(1n+8) {
+  margin: 0 0 0 10px;
+}
+
+.nav__link :nth-child(1n+2):not(:last-child) {
+  margin: 0 10px;
+}
```

Comment:
same, simplify specificity

Reply:
could you give me some example for understanding please. I did that according to the lessons which i saw. I would be grateful if you could tell me how to do this more simple.